libs/core/src/components/pds-radio/pds-radio.tsx

```diff
@@ -0,0 +1,114 @@
+import { Component, Host, h, Prop, Event, EventEmitter } from '@stencil/core';
+import { assignDescription, messageId } from '../../utils/form';
+
+@Component({
+  tag: 'pds-radio',
+  styleUrl: 'pds-radio.scss',
+  scoped: true,
+})
+export class PdsRadio {
+  /**
+   * Determines whether or not the radio is checked.
+   * @defaultValue false
+   */
+  @Prop() checked = false;
+
+  /**
+   * Determines whether or not the radio is disabled.
+   * @defaultValue false
+   */
+  @Prop() disabled = false;
+
+  /**
+   * Determines whether or not the radio is invalid.
+   * @defaultValue false
+   */
+  @Prop() invalid = false;
+
+  /**
+   * String used for radio `id` attribute and label `for` attribute.
+   */
+  @Prop() componentId: string;
```

Comment:
Can this prop be updated to be required so it's content is used in the input id?